2022-09-20 15:51:43,599 - app.py[line:142] - INFO: 你有够恶心的乐
2022-09-20 15:51:43,621 - app.py[line:154] - INFO: AI handle
2022-09-20 15:51:44,035 - app.py[line:164] - INFO: {'好': 0, '乐': 0, '哀': 0, '怒': 0, '惧': 0, '恶': 1, '惊': 0}
2022-09-20 15:51:44,035 - app.py[line:176] - ERROR: Attempt to convert a value (None) with an unsupported type (<class 'NoneType'>) to a Tensor.
2022-09-20 15:52:53,663 - app.py[line:134] - INFO: submit
2022-09-20 15:52:55,342 - app.py[line:56] - INFO: download successful
2022-09-20 15:52:55,342 - app.py[line:138] - INFO: ./img/user/Cyivwpq.jpg
2022-09-20 15:52:55,343 - app.py[line:142] - INFO: 你有够恶心的乐
2022-09-20 15:52:55,363 - app.py[line:154] - INFO: AI handle
2022-09-20 15:52:55,364 - app.py[line:164] - INFO: {'好': 0, '乐': 0, '哀': 0, '怒': 0, '惧': 0, '恶': 1, '惊': 0}
2022-09-20 15:53:08,512 - app.py[line:171] - INFO: {'0': 'https://i.imgur.com/7fiQ1CE.jpeg'}
2022-09-20 15:56:10,760 - app.py[line:183] - INFO: initialize success
2022-09-20 15:56:12,817 - app.py[line:134] - INFO: submit
2022-09-20 15:56:14,498 - app.py[line:56] - INFO: download successful
2022-09-20 15:56:14,498 - app.py[line:138] - INFO: ./img/user/Cyivwpq.jpg
2022-09-20 15:56:14,499 - app.py[line:142] - INFO: 你有够恶心的乐
2022-09-20 15:56:14,513 - app.py[line:154] - INFO: AI handle
2022-09-20 15:56:14,929 - app.py[line:164] - INFO: {'好': 0, '乐': 0, '哀': 0, '怒': 0, '惧': 0, '恶': 1, '惊': 0}
2022-09-20 15:56:28,448 - app.py[line:171] - INFO: {'0': 'https://i.imgur.com/DAtwseR.jpeg', '1': 'https://i.imgur.com/jRusgPL.jpeg'}
2022-09-20 16:08:49,569 - app.py[line:197] - INFO: initialize success
2022-09-20 16:08:52,318 - app.py[line:143] - INFO: submit
2022-09-20 16:08:54,059 - app.py[line:56] - INFO: download successful
2022-09-20 16:08:54,059 - app.py[line:147] - INFO: ./img/user/Cyivwpq.jpg
2022-09-20 16:08:54,060 - app.py[line:151] - INFO: 你有够恶心的乐
2022-09-20 16:08:54,074 - app.py[line:163] - INFO: AI handle
2022-09-20 16:08:54,564 - app.py[line:173] - INFO: {'好': 0, '乐': 0, '哀': 0, '怒': 0, '惧': 0, '恶': 1, '惊': 0}
2022-09-20 16:08:54,564 - app.py[line:65] - INFO: AI upload
2022-09-20 16:08:54,582 - app.py[line:94] - INFO: Start transfer
2022-09-20 16:09:00,244 - app.py[line:96] - INFO: Start upload
2022-09-20 16:09:03,191 - app.py[line:98] - INFO: End num 0
2022-09-20 16:09:03,211 - app.py[line:94] - INFO: Start transfer
2022-09-20 16:09:05,167 - app.py[line:96] - INFO: Start upload
2022-09-20 16:09:08,239 - app.py[line:98] - INFO: End num 1
2022-09-20 16:09:08,239 - app.py[line:180] - INFO: {'0': 'https://i.imgur.com/NyDlBWX.jpeg', '1': 'https://i.imgur.com/I1xE95F.jpeg'}
2022-09-20 17:00:03,052 - app.py[line:143] - INFO: submit
2022-09-20 17:00:03,053 - app.py[line:59] - ERROR: download failed
2022-09-20 17:00:03,054 - app.py[line:60] - ERROR: <urlopen error unknown url type: {"data">
2022-09-20 17:00:03,054 - app.py[line:147] - INFO: None
2022-09-20 17:00:03,055 - app.py[line:151] - INFO: rrtgfg
2022-09-20 17:00:03,169 - app.py[line:163] - INFO: AI handle
2022-09-20 17:00:03,170 - app.py[line:173] - INFO: {'好': 0, '乐': 0, '哀': 0, '怒': 0, '惧': 0, '恶': 0, '惊': 0}
2022-09-20 17:00:03,170 - app.py[line:65] - INFO: AI upload
2022-09-20 17:00:03,172 - app.py[line:103] - ERROR: Ai error: Attempt to convert a value (None) with an unsupported type (<class 'NoneType'>) to a Tensor.
2022-09-20 17:00:03,172 - app.py[line:180] - INFO: None
2022-09-20 17:00:03,176 - app.py[line:189] - ERROR: 500 Internal Server Error: The server encountered an internal error and was unable to complete your request. Either the server is overloaded or there is an error in the application.
2022-09-20 17:01:56,896 - app.py[line:143] - INFO: submit
2022-09-20 17:01:56,897 - app.py[line:59] - ERROR: download failed
2022-09-20 17:01:56,897 - app.py[line:60] - ERROR: <urlopen error unknown url type: {"data">
2022-09-20 17:01:56,898 - app.py[line:147] - INFO: None
2022-09-20 17:01:56,899 - app.py[line:151] - INFO: dgdhxh
2022-09-20 17:01:57,026 - app.py[line:163] - INFO: AI handle
2022-09-20 17:01:57,027 - app.py[line:173] - INFO: {'好': 0, '乐': 0, '哀': 0, '怒': 0, '惧': 0, '恶': 0, '惊': 0}
2022-09-20 17:01:57,027 - app.py[line:65] - INFO: AI upload
2022-09-20 17:01:57,028 - app.py[line:103] - ERROR: Ai error: Attempt to convert a value (None) with an unsupported type (<class 'NoneType'>) to a Tensor.
2022-09-20 17:01:57,029 - app.py[line:180] - INFO: None
2022-09-20 17:01:57,030 - app.py[line:189] - ERROR: 500 Internal Server Error: The server encountered an internal error and was unable to complete your request. Either the server is overloaded or there is an error in the application.
2022-09-20 17:06:10,481 - app.py[line:143] - INFO: submit
2022-09-20 17:06:10,483 - app.py[line:59] - ERROR: download failed
2022-09-20 17:06:10,484 - app.py[line:60] - ERROR: <urlopen error unknown url type: {"data">
2022-09-20 17:06:10,484 - app.py[line:147] - INFO: None
2022-09-20 17:06:10,485 - app.py[line:151] - INFO: xhchc
